tetris.py

```diff
@@ -192,5 +196,16 @@ def block_rotator(self, event):
             rotate.append((y, x))
         self.current_block = rotate
 
+    def full_line_clear(self):
+        y_coordinates = [y for (x, y) in self.landed_blocks]
+        coordinates_counter = collections.Counter(y_coordinates)
+        print(coordinates_counter)
+        for y_line, count in coordinates_counter.items():
+            if count == game_width:
+                self.landed_blocks = [(a, b) for (a, b) in self.landed_blocks if b != y_line]
```

Comment:
This lines deletes the blocks on the `y_line`, but that's not necessary, because the 1km line below deletes them too (it does not delete blocks with `b > y_line` or `b < y_line`, and deletes other blocks, those with `b == y_line`).